satpy/tests/reader_tests/test_satpy_cf_nc.py

```diff
@@ -120,3 +120,93 @@ def test_fix_modifier_attr(self):
         ds_info = {'modifiers': []}
         self.reader.fix_modifier_attr(ds_info)
         self.assertEqual(ds_info['modifiers'], ())
+
+    def test_read_prefixed_channels(self):
```

Comment:
This test function should be split into 2 smaller one for cleanliness. From there, I guess there will be parts that can be factorized so as to avoid code duplication :)

Reply:
ok